src/screens/HomeScreen.js

```diff
@@ -13,31 +13,34 @@ const HomeScreen = ({ navigation, route }) => {
 
   return (
     <View style={styles.container}>
-      <ButtonComponent  
-        title="Go to Habit options" 
-        onPress={() => navigation.navigate('Habitoptions')} 
-      />
-      {selectedHabit && (
+            <ButtonComponent onPress={() => navigation.navigate('Habitoptions')}>
+              <Text style={styles.plusStyle}>+</Text>
+              <Text style={styles.buttonTextStyle}></Text>
+              <Text style={styles.buttonTextStyle}>New Habit</Text>
+            </ButtonComponent>
+            {selectedHabit && (
         <Text style={styles.selectedHabitText}>
           Selected Habit: {route.params?.selectedHabit}
         </Text>
       )}
-    </View>
+        </View>
```

Comment:
you can better indent these lines so that they look better in code

Reply:
Indent is fixed.